conans/test/update_settings_yml_test.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+from conans.test.tools import TestClient
+from conans.util.files import load, save
+
+
+class UpdateSettingsYmlTest(unittest.TestCase):
+    """ This test is to validate that after adding a new settings, that allows a None
```

Comment:
This test might be useless, as it was checking the upgrade from no setting to having sub-setting with value=None